### DISCUSSION TITLE
2023년을 보내고 2024년을 맞이하며 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#57] [Announcements]
# 2023년을 보내고 2024년을 맞이하며 - 정현수 기술 블로그

내가 무엇을 했는가에 대한 기록이자, 원동력을 위한 자기 칭찬이자, 앞으로 어떻게 나아가야 하는지에 대한 이정표이다.

https://junghyeonsu.com/posts/2023-retrospect/

**Comment by haryan248** [comment]
테오콘때 발표 듣고 많이 느꼈습니다.
한해동안 고생하셨습니다 :)

**Reply by junghyeonsu** [reply]
감사합니다!
